eng/pipelines/stress/http-linux.yml

```diff
@@ -0,0 +1,28 @@
+trigger: none
+
+pool:
+  name: Hosted Ubuntu 1604
+
+variables:
+  httpStressProject: src/System.Net.Http/tests/StressTests/HttpStress
+  sdkBaseImage: sdk-corefx-current
+  httpStressImage: httpstress
+
+steps:
+- checkout: self
+  clean: true
+  fetchDepth: 0
+  lfs: false
+
+- bash: |
+    docker build -t $(sdkBaseImage) -f $(Build.SourcesDirectory)/$(HttpStressProject)/corefx.Dockerfile .
```

Comment:
I'm might be missing something but I don't fully understand how this is building corefx.

Reply:
Disregard, I see there are 2 different dockerfiles.